metastore/src/main/scala/com/gu/tableversions/metastore/VersionPaths.scala

```diff
@@ -31,8 +32,17 @@ object VersionPaths {
       partitionPath
     else {
       def normalised(path: String): String = if (path.endsWith("/")) path else path + "/"
+
       def versioned(path: String): String = s"$path${version.label}"
+
       new URI(versioned(normalised(partitionPath.toString)))
     }
 
+  def flattenMap(partitions: List[Partition]): Map[String, String] = {
+    import scala.language.postfixOps
```

Comment:
We tend not to use this in other parts of this codebase so I'd go with plain `.map` here as well.

Reply:
Changed